src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,28 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
             return;
         }
         OrderService orderService = new OrderServiceImpl();
         orderService.completeOrder(user.getUserId());
+
+        try {
+            user = authenticationService.login("boben", "12345");
+        } catch (AuthenticationException e) {
+            return;
```

Comment:
As with the previous catch block, it's important to log the exception to have a clearer understanding of what went wrong. Use logger.error or logger.warn with a descriptive message.